docs/architecture/adr-063-core-module-api.md

```diff
@@ -414,6 +414,51 @@ should check this compatibility version and return an error if the current `Runt
 than the version of the core API that this runtime version can support. When new features are adding to the `core`
 module API that runtime modules are required to support, this version should be incremented.
 
+### Runtime Modules
+
+The initial `runtime` module will simply be created within the existing `github.com/cosmos/cosmos-sdk` go module
+under the `runtime` package. This module will be a small wrapper around the existing `BaseApp`, `sdk.Context` and
+module manager and follow the Cosmos SDK's existing [0-based versioning](https://0ver.org). To move to semantic
+versioning as well as runtime modularity, new officially supported runtime modules will be created under the
+`cosmossdk.io/runtime` prefix. For each supported consensus engine a semantically-versioned go module should be created
+with a runtime implementation for that consensus engine. For example:
+- `cosmossdk.io/runtime/comet`
+- `cosmossdk.io/runtime/comet/v2`
+- `cosmossdk.io/runtime/rollkit`
+- etc.
+
+These runtime modules should attempt to be semantically versioned even if the underlying consensus engine is not. Also,
```

Comment:
Why is runtime considered an SDK module? What's your definition of an SDK module?

Reply:
Based on the definition in ADR 057, a module is anything that is loadable as a module. And runtime is a module because it could allow configuration of a whole app (including upgrading consensus, begin/end blocker ordering, etc.) to be updated based just on the app config. See the explanation given for this in ADR 057: https://github.com/cosmos/cosmos-sdk/blob/851e9e873b5c68aefe098b78f1a23d5acf50a81f/docs/architecture/adr-057-app-wiring.md?plain=1#L102-L132